src/core/js/scriptLoader.js

```diff
@@ -34,24 +45,27 @@
 
     // 3. start loading JQuery, wait for it to be loaded
     function loadJQuery() {
-        Modernizr.load([
-            {
-                load: 'libraries/jquery.v2.min.js',
-                complete: checkJQueryStatus
-            }
-        ]);
+        loadScript('libraries/jquery.v2.min.js', checkJQueryStatus);
```

Comment:
Isn't this altered by the switch to JQuery v3?

Reply:
Yup, but they are two separate PRs, so I'll hit that merge conflict when I get there,